test/open-patchsafe-test.js

```diff
@@ -14,7 +14,7 @@ function test (fun) {
   return function (done) {
     var location = common.nextLocation()
     // 1) open database without callback, opens in worker thread
-    var db = levelup(leveldown(location), { valueEncoding: 'utf8' })
+    var db = levelup(leveldown(location)/*, { valueEncoding: 'utf8' }*/)
 
```

Comment:
Just remove `/* .. */` instead or wrap with `encoding-down` to be explicit (apparently not needed)

Reply:
Also `standard` complains :wink:

Reply:
cleaned up and combined commits to date.